pkg/component/monitoring/charts/bootstrap/templates/aggregate-prometheus/config.yaml

```diff
@@ -67,6 +67,7 @@ data:
         - '{__name__=~"metering:.+", __name__!~"metering:.+(over_time|_seconds|:this_month)"}'
         - '{__name__=~"seed:(.+):(.+)"}'
         - '{job="kube-state-metrics",namespace=~"garden|extension-.+"}'
+        - '{job="kube-state-metrics",namespace=""}'
```

Comment:
Can we merge both in one?

Reply:
that was an idea, but it was not very easy to read / understand for an untrained eye. It would have looked like 

`- '{job="kube-state-metrics",namespace=~"garden|extension-.+|"}'`

Having it explicitly on a new line, would enhance readability, I believe.

Reply:
Ok. 👍 
/approve